WSRequestHandler.cpp

```diff
@@ -401,8 +413,90 @@ void WSRequestHandler::HandleStartStreaming(WSRequestHandler* req)
 {
 	if (obs_frontend_streaming_active() == false)
 	{
+		obs_data_t* streamData = obs_data_get_obj(req->data, "stream");
```

Comment:
In `rtmp-settings`, that behaviour was kinda mandatory because there wasn't an other way to set streaming settings before starting it. Now that we have the proper methods in the frontend API, there's not need to implement that behavior. A user willing to set streaming settings should call `SetStreamSettings` followed by `StartStreaming`.